docs/resources/cluster_v2.md

```diff
@@ -58,6 +58,44 @@ resource "rancher2_machine_config_v2" "foo" {
   }
 }
 
+# Create a new rancher v2 Cluster with multiple machine pools
+resource "rancher2_cluster_v2" "foo-rke2" {
+  name = "foo-rke2"
+  kubernetes_version = "v1.21.4+rke2r2"
```

Comment:
Update to `kubernetes_version = "<RANCHER_KUBERNETES_VERSION"` since this is the standard now. Otherwise, we'll need to maintain version number.

Reply:
Done